spec/support/feature_macros.rb

```diff
@@ -0,0 +1,9 @@
+module FeatureMacros
+  def log_in(user = nil)
+    user = user || Fabricate(:user)
```

Comment:
@tomtomecek  
True, I originally had it this way.  But I noticed that Kevin seems to prefer the other way sometimes in his videos.  Maybe those were different cases.